src/api/eth.rs

```diff
@@ -366,14 +422,16 @@ mod tests {
     Eth:block:block_by_hash, BlockId::Hash(0x123.into()), true
     =>
     "eth_getBlockByHash", vec![r#""0x0000000000000000000000000000000000000000000000000000000000000123""#, r#"true"#];
-    Value::Null => ()
+    ::serde_json::from_str(EXAMPLE_BLOCK).unwrap()
+    => ::serde_json::from_str::<Block>(EXAMPLE_BLOCK).unwrap()
```

Comment:
but then we'd also have to serialize the default struct to value -- not a huge win, but you might be right